sdk/src/main/scala/com/horizen/consensus/TimeToEpochSlotConverter.scala

```diff
@@ -3,13 +3,39 @@ package com.horizen.consensus
 import com.horizen.params.NetworkParams
 import scorex.core.block.Block
 
+
+case class EpochAndSlot(epochNumber: ConsensusEpochNumber, slotNumber: ConsensusSlotNumber) extends Comparable[EpochAndSlot] {
```

Comment:
`EpochAndSlot` -> `ConsensusEpochAndSlot` to separate from withdrawal epoch and slot